index.js

```diff
@@ -0,0 +1,42 @@
+import renderBooks from './modules/renderer.js';
+import Books from './modules/class.js';
+import { DateTime } from './node_modules/luxon/build/es6/luxon.js';
+
```

Comment:
- [ ] Optional suggestion

How about doing this, to import from luxon `import { DateTime } from "luxon"; ` instead of `'./node_modules/luxon/build/es6/luxon.js'.`